pkg/pligos/make.go

```diff
@@ -0,0 +1,106 @@
+package pligos
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+	"realcloud.tech/pligos/pkg/compiler"
+	"realcloud.tech/pligos/pkg/pathutil"
+)
+
+func MakeCompiler(config PligosConfig, contextName string) (*compiler.Compiler, error) {
```

Comment:
Why MakeCompiler receive a PligosConfig and MakeCreateConfig builds a PligosConfig from a pligosPath? This is inconsistent.